config/variables/stacks/appeals-service/test.hcl

```diff
@@ -1,13 +1,5 @@
 locals {
   appeals_feature_flags = [
-    {
-      name    = "appeal-form-v2"

```

Comment:
might be sensible to separate this into a separate PR as a draft, need to wait until the front end commit is released before merging this in

Reply:
Good call, will reinstate on this PR